### ISSUE TITLE
Broke deployment

### ISSUE THREAD

**Issue by bradleyden** [ga-wdi-boston/capstone-project#880]
I made a few adjustments from my most recent deployment to fix a couple things, and I am not able to redeploy again, I'm getting site not found when I go to the url even though it seems to have built and pushed to the branch without a problem. I think something weird is going on with my dist/ files and gitignore that's making the process not work properly.

**Comment by jordanallain** [comment]
What were your redeployment steps? Did you get any errors in your terminal? Did you change anything major before redeploying?

**Comment by bradleyden** [comment]
Sorry, I worked it out with Chris right as we were being kicked out of the classroom for a class so I didn't have a chance to close the issue, but it is working now. Unfortunately I don't really know what Chris did differently because we were so rushed so I can't really explain what the issue was or how it was fixed, so it must have just been Thursday of capstone fatigue making me miss something. I will open a higher quality issue and reach out again if I have issues with deploying in the future, but I won't be messing with it again before my presentation tomorrow. Thanks for the response anyway!
